### ISSUE TITLE
Bloquer les animations sur mobile iOS

### ISSUE THREAD

**Issue by MATFUN** [pojome/elementor#5945]
@media only screen and (max-width : 480px) 
{ .animated { animation: none !important;  
} 
}  

–––––
This CSS code works for Android mobiles, but not for iOS mobiles.
It should be added a possibility to do this directly in an Elementor parameter.

**Comment by bainternet** [comment]
Thanks for reporting,
Please follow guidelines for opening a new [issue report](https://github.com/pojome/elementor/issues/new?template=Bug_report.md), so we can help you better.
This report is missing key information: 

* Prerequisites.
* Steps to replicate.
* Isolating the problem.
* System info.
* Should be in plain English!
